### ISSUE TITLE
Комментарии к первоначальной заявке

### ISSUE THREAD

**Issue by aakuptsov** [AlisMori/IVR_Medications-compatibility#1]
1. Необходимо заполнить все поля и в 2359
2. Из описания в разделах [ ПОХОЖИЕ / АНАЛОГИЧНЫЕ ПРОДУКТЫ ] неясно, почему упомянутые продукты не решают проблемное поле. Также нужно добавить 3-ий продукт (вместо "другие приложения")
3. Нужно описать (в функциональных требованиях), как лекарства будут попадать в приложения.
4. А нет ли похожих сайтов для решения проблемного поля?

**Comment by aakuptsov** [comment]
5. Функциональных требований должно быть минимум 7. 

6. Просьба уточнить, что значит "вручную": через админку? Хардкодить лекарство в приложении категорически нельзя.

**Comment by AlisMori** [comment]
> 1. Необходимо заполнить все поля и в 2359
> 2. Из описания в разделах [ ПОХОЖИЕ / АНАЛОГИЧНЫЕ ПРОДУКТЫ ] неясно, почему упомянутые продукты не решают проблемное поле. Также нужно добавить 3-ий продукт (вместо "другие приложения")
> 3. Нужно описать (в функциональных требованиях), как лекарства будут попадать в приложения.
> 4. А нет ли похожих сайтов для решения проблемного поля?

**Comment by AlisMori** [comment]
2. Похожие и аналогичные продукты представляют собой сайты, что неудобно для ежедневного пользования, или имеют только часть функций, которые будут реализованы в моём продукте, представляющий собой мобильное приложение. 
4. Как было упомянуто выше, сайты слегка неудобны для ежедневного использования, для поиска информации необходимо выполнить больше действий, нежели искать её в специализированном приложении.
5. В первом пункте функциональных требований указано сразу три аналогичные возможности продукта
6. Более точное описание создания базы данных было  добавлено в саму заявку.

**Comment by aakuptsov** [comment]
>>Похожие и аналогичные продукты представляют собой сайты


Так, погоди, Medisafe и MyTherapy - это же приложения, как ты сама пишешь в заявке. Плюс не сказано, какие у них недостатки. Добавь конкретные недостатки, пожалуйста (это в будущем пригодится и на защите).


>>В первом пункте функциональных требований указано сразу три


А ты их разбей, пожалуйста, на два пункта и будет семь. В оформлении любой проектной документации формальные требования, увы, важны не менее, чем содержательные (мы поэтому и обращаем на них такое внимание, потому что это не только на наших проектах, а в общем-то везде).

**Comment by AlisMori** [comment]
> Плюс не сказано, какие у них недостатки.

Конкретизировала недостатки

**Comment by pvs9** [comment]
Исходя из инструментов разработки не вижу заложенного стэка под бэкенд. Приложения без бэкенда принять не можем

**Comment by AlisMori** [comment]
> Исходя из инструментов разработки не вижу заложенного стэка под бэкенд.

Конкретизировала инструменты разработки
